Cargo.toml

```diff
@@ -21,7 +21,7 @@ sd-notify = { version = "0.1.1" }
 toml = "0.4.2"
 serde = { version = "1.0", features = ["derive"] }
 env_logger = "0.7.1"
-log = "0.4.8"
+log = { version = "0.4.8", features = ["serde"] }
```

Comment:
Just for my curiosity: why is the `serde` feature needed here?

Reply:
https://github.com/parallaxsecond/parsec/pull/51/files#diff-5c0f3ed7e332f3aead779609ed270a50R53

The config key that we're deserializing from the config file is essentially a log level filter (which has to have the serde feature enabled for that)